Icebnb/Icebnb/RangeSliderWidget.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CosmosViewWidget.swift
+//  ecommerce
+//
+//  Created by Guy Daher on 6/16/17.
+//  Copyright © 2017 Guy Daher. All rights reserved.
```

Comment:
nit: Guy -> Robert. Or maybe we should change all of those to Algolia or something?

Reply:
I used it from the ecommerce project, so the copyright should belong to you. But indeed, we should use an Algolia signature instead